### ISSUE TITLE
PSR12.ControlStructures.BooleanOperatorPlacement false positive when operator is the only content on line

### ISSUE THREAD

**Issue by morozov** [squizlabs/PHP_CodeSniffer#3075]
**Describe the bug**
A fixable rule violation is cannot be auto-fixed.

**Code sample**
```php
<?php

function test($value)
{
    if (
        ($value == 1 ||
            $value == 2)
        &&
        ($value == 3 ||
            $value == 4)
    ) {
        return 5;
    }

    return 6;
}
```

**Custom ruleset**
None.

**To reproduce**
1. Run `phpcs -s --standard=PSR12 test.php`
```
FILE: /home/morozov/Projects/phpcs-playground/test.php
------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------
 5 | ERROR | [x] Boolean operators between conditions must be at the beginning or end of the
   |       |     line, but not both
   |       |     (PSR12.ControlStructures.BooleanOperatorPlacement.FoundMixed)
------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------

Time: 44ms; Memory: 6MB
```
2. Run `phpcbf --standard=PSR12 test.php`
```
PHPCBF RESULT SUMMARY
----------------------------------------------------------------------
FILE                                                  FIXED  REMAINING
----------------------------------------------------------------------
/home/morozov/Projects/phpcs-playground/test.php      FAILED TO FIX
----------------------------------------------------------------------
A TOTAL OF 0 ERRORS WERE FIXED IN 1 FILE
----------------------------------------------------------------------
PHPCBF FAILED TO FIX 1 FILE
----------------------------------------------------------------------

Time: 46ms; Memory: 6MB
```

**Expected behavior**
The violation is fixed.

**Versions (please complete the following information):**
 - OS: Linux
 - PHP: 7.4.9
 - PHPCS: 3.5.6
 - Standard: Squiz

**Comment by gsherwood** [comment]
Looks to be an issue caused by the operator being the only content on the line. There is nothing to fix here as all operators are the last content on the line, so the error shouldn't be showing at all.

**Comment by gsherwood** [comment]
Thanks for reporting this. A fix has been committed and will be released in version 3.5.7
